addons/web/static/src/js/views/calendar/calendar_renderer.js

```diff
@@ -451,7 +451,7 @@ return AbstractRenderer.extend({
                 var view = info.view;
                 self._addEventAttributes(element, event);
                 if (view.type === 'dayGridYear') {
-                    const color = this.getColor(event.extendedProps.color_index);
+                    const color = self.getColor(event.extendedProps.color_index);
```

Comment:
Not sure where to note this comment as it is related to the commit '[FIX] web: this undefined in fullCalendarOptions' message 🙂.

I think it would be best to use `this` in the commit message as I did not get the sense of it as this is not 'ceci' but `this` in the code 😄.